ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2104 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Machine Learning for the 20th century: <br> Can a United Kingdom database from 1987 classify a glass cosmetics jar from the 1930s that might have belonged to Amelia Earhart?\n",
+    "Author: Joe Cerniglia <br>\n",
+    "Reviewers: <br>\n",
+    "Date: March 18, 2022 <br><br>\n",
+    "\n",
+    "## Background\n",
+    "In the summer of 2010, researchers from The International Group for Historic Aircraft Recovery excavated, in fragments, a small semi-opaque cosmetic jar from the Pacific island of Nikumaroro. They were searching for evidence to support their hypothesis that Amelia Earhart and her navigator Fred Noonan perished there in 1937 after very nearly succeeding in their pathbreaking mission to circumnavigate the globe by air at its widest point, the equator.\n",
+    "\n",
+    "My interest in glass artifacts from the island ultimately led to a personal quest to become a resident glass expert of the team. In partnership with archaeologists Thomas King and Bill Lockhart, and chemist Greg George, we produced a [paper](https://www.academia.edu/40823470/A_Freckle_In_Time_or_a_Fly_in_the_Ointment) that summarized our findings. By then, our work on the jar had already generated some [press](https://www.nbcnews.com/id/wbna47623025).\n",
+    "\n",
+    "While researching the jar in 2013, I requested EAG Labs of Sunnyvale, California to determine the chemical composition of the artifact jar, and that of a 'sibling' glass jar that I had purchased on eBay and had named the clear facsimile. The clear facsimile jar was similar to the artifact in most respects, but importantly, it was transparent while the artifact was semi-opaque. As their [report](https://tighar.org/Projects/Earhart/Archives/Research/ResearchPapers/freckleintime/Document_02_Facsimileclearjarreport.pdf) shows, the two jars turned out to have very different chemical compositions. Interestingly, both jars were manufactured by the same company, Hazel-Atlas of Wheeling, West Virginia.\n",
+    "\n",
+    "\n",
+    "## Data \n",
```

Comment:
Suggestions for making the copy a bit more concise in the Data section:

In the social sciences, "data" often means a collection of facts. In industry, however, we often consider data to be a collection of numbers and vectors, which we use computational tools like statistics and machine learning to analyse. As part of an eCornell machine learning course, I had the opportunity to rekindle my work on the jars and utilise the EAG labs dataset for a machine learning classification problem.

In machine learning, classification algorithms sort data into pre-defined classes by analysing various features, or coordinate plots on a multi-dimensional plane, of the data. Each data point has its own unique set of features. The jars, and in this case the glass materials of the jars, offered three key measurements which represented our features: calcium, magnesium, and barium. While the EAG labs dataset was sufficient for our previous analysis, it was not well suited for a machine learning application, which typically require many examples of each class in a classification problem in order to "train" the algorithm to classify new data it has not seen before. This need for more data presented a challenge: where would I find the necessary data to effectively train a machine learning model?

I did not need to search very far for my answer. British forensic scientists Ian W. Evett and Ernest J. Spiehler assembled a database of glass samples in 1987 for the purposes of solving crimes, such as break-ins. Their paper was first presented at the 1987 conference of the KBS (Knowledge-Based Systems) in Goverment and is titled:
"Rule Induction in Forensic Science." http://gpbib.cs.ucl.ac.uk/gp-html/evett_1987_rifs.html. The database is available at the University of California Machine Learning Repository [here](https://archive.ics.uci.edu/ml/datasets/Glass+Identification). The features of their data comprised of the weight percent of eight elemental components as well as the refractive index. The types of glass represented in the sample are: Window Float, Window Non-Float, Vehicle Float, Container, Tableware, and Headlamp. There are no Vehicle Non-Float types represented in the data.

In comparison to this dataset, my EAG labs data was independently measured for most of the same elements as found in the U.K. database, with a few caveats, which required additional domain-specific processing in order to align the two datasets:
- Because silicon was measured in my lab data as 'Matrix,' with no actual number stated, I estimated the silicon for both containers based on the average for containers in the database (72.37). 
- Because K (potassium) was measured at 980 ppm for the clear facsimile, rather than by wt%, I assigned it a value of 50% of the wt% for the artifact jar: .5 X .24 = .12. 
- Because Fe (iron) is at very low wt% levels in the 1987 database, and the levels in my data are listed at very low parts per million (ppm), I assigned to the artifact jar and to the clear facsimile a wt% of Fe of .02 and .01, respectively.
- Refractive index was not measured for either the artifact jar or the clear facsimile. Since the clear facsimile is completely transparent, I assigned it the minimum refractive index of containers from the 1987 dataset. Since the artifact jar is semi-opaque, I assigned it the maximum refractive index of containers from the 1987 dataset.

These educated guesses were informed by my background in this space and were necessary to prepare the appropriate dataset for the machine learning problem demonstrated in this notebook.